.circleci/config.yml

```diff
@@ -45,6 +50,21 @@ jobs:
           paths:
             - build/*
             - s3_website.yml
+  build_pdf:
+    working_directory: ~/work
+    docker:
+      - image: tianon/latex
+    steps:
+      - checkout
```

Comment:
Instead of `checkout`, which checks out the git repo, you'll need to attach the workspace you persisted earlier. That workspace has `build/*` which is what you need.

```yaml
- attach_workspace:
    at: ~/work
```